### DISCUSSION TITLE
How to update EmailValidator dependency to v3

### DISCUSSION THREAD

**Discussion by egulias** [laravel/framework#38390] [General]
Hello Laravel community, 
I'm Eduardo, author & maitainer of the [EmailValidator library](https://github.com/egulias/EmailValidator).
At the beggining of 2021 v3 was released, followed by v3.1, requiring PHP 7 and benefiting from the new features with a complete refactor and some improvements from users feedback (mainly around domain part validation).
I have come to realize that Laravel is still using v2. **Version 2 will EOL in 2022** (in roughly 4 months).

I don't have enough knowledge of Laravel, can anyone point me to which points any BC changes might be triggered? Any other place besides `framework`'s composer.json to update?

Thanks for your help!

**Comment by driesvints** [comment]
Support will come in Laravel v9: https://github.com/laravel/framework/pull/36525

**Comment by egulias** [comment]
Awesome! Let me know if you face any issue where I can help.
